matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -109,7 +109,7 @@ internal class EventRelationsAggregationProcessor @Inject constructor(@UserId pr
                 return
             }
             val isLocalEcho = LocalEcho.isLocalEchoId(event.eventId ?: "")
-            when (event.getClearType()) {
+            when (event.type) {
```

Comment:
I think i really wanted the clear type here.
If clear type was encrypted, i would try to see from non encrypted data if there is something i can do. 
For example if i sent a verification request, and i receive an accept that i can't decrypt I can cancel

Reply:
I checked, we should keep type. It should not have been changed